auto_submit/lib/service/validation_service.dart

```diff
@@ -368,6 +372,41 @@ Exception: ${exception.message}
     return ProcessMergeResult.noMessage(true);
   }
 
+  /// A user can provide an alternate merge method via comment by providing the
+  /// string @autosubmit:<merge_method>.
+  static final RegExp regExpMergeMethod = RegExp(r'@autosubmit:(merge|squash|rebase)', caseSensitive: false);
+  Future<github.MergeMethod> checkForMergeMethod({
+    required github.RepositorySlug slug,
+    required int prNumber,
+  }) async {
+    const github.MergeMethod defaultMergeMethod = github.MergeMethod.squash;
+    final GithubService githubService = await config.createGithubService(slug);
+
+    final List<github.IssueComment> issueComments = await githubService.listIssueComments(slug, prNumber);
```

Comment:
Do we need to check who the author of the comment is?

Reply:
Ah yes!!! Totally forgot this. Good catch.

Reply:
Going to have to make a change in the github repo.

Reply:
Done!